packages/block-editor/src/components/link-control/link-preview.js

```diff
@@ -99,16 +97,14 @@ export default function LinkPreview( {
 					<span className="block-editor-link-control__search-item-details">
 						{ ! isEmptyURL ? (
 							<>
-								<Tooltip text={ value.url }>
-									<ExternalLink
-										className="block-editor-link-control__search-item-title"
-										href={ value.url }
-									>
-										<Truncate numberOfLines={ 1 }>
-											{ displayTitle }
-										</Truncate>
-									</ExternalLink>
-								</Tooltip>
+								<ExternalLink
+									className="block-editor-link-control__search-item-title"
+									href={ value.url }
+								>
+									<Truncate numberOfLines={ 1 }>
+										{ displayTitle }
+									</Truncate>
+								</ExternalLink>
```

Comment:
I recommend we don't remove the tooltip. Why? Because histroically this has been required to allow users to see the full URL. 

This is necessary, for example, when you link to a file or an image which can have very long URLs.

Prior rationale for introducing this can be found in [this discussion](https://github.com/WordPress/gutenberg/pull/54920#issuecomment-1746384383).

In my opinion this change, as much as it seems like a nice "tidy", would be a regression.

Reply:
The problem with having the tooltip is that focus is moved to it after link creation which reveals the tooltip and:
1. Covers the action buttons if it's a long URL
2. Obstructs the identical url text beneath it if it's a short URL

<img width="398" alt="Screenshot 2024-01-29 at 3 57 48 PM" src="https://github.com/WordPress/gutenberg/assets/967608/f31127ec-18aa-43b3-9cdd-01f61a42d3bb">

If the full url value is needed from the preview and we also don't want to display too much of the URL, then we could:
- Have the copy button tooltip include the full url, ie: "Copy: [URL value]"
- Have a button or tooltip icon next to truncated URLs that can be used to access the full value

Reply:
Understood. Thanks for illustrating that 👍 

There is perhaps an argument around the copy button. How about we move the tooltip to the `Copy` button? 

I'm open to any suggestion really.

I wanted to make sure we understood that having a tooltip was an intentional UX decision to alleviate real-world user feedback and so we shouldn't just remove it without providing a solid alternative.

I appreciate this was easy to miss though 🙇

Reply:
I don't think we implemented the suggestion here https://github.com/WordPress/gutenberg/pull/58183/files#r1471501443

Reply:
I'm not sure about my comment's validity there. I think it might be fine for it to be "Copy: [URL Value]."

Reply:
I'll add the URL to the Copy tooltip and see how it feels.

Reply:
https://github.com/WordPress/gutenberg/pull/58183/commits/0a6d92c6e12eb5bfe75c355e6c22e96d4d7d18fb

Reply:
I tried using `sprintf` which I hope makes it clearer and is still legit. Not 100% sure though 😓